.gitignore

```diff
@@ -20,3 +20,4 @@ settings.xml
 
 
 src/scratch
+dev_tools/
```

Comment:
After looking through, I realize this was for the jar files that get downloaded?

If so, I don't think we should include those scripts at this time and thus we won't need to ignore this dir.